### ISSUE TITLE
Commits from PRs #140 and #143 are not visible on develop branch

### ISSUE THREAD

**Issue by shoshijak** [cp2k/dbcsr#145]
I merged PRs #140 and #143 to `develop` 2 days ago but cannot see the commits of these PRs in develop ... any idea why that is?

**Comment by shoshijak** [comment]
I've pushed the branches corresponding to these PRs back to my fork of DBCSR in case:
- [branch](https://github.com/shoshijak/dbcsr/tree/fix/perf_issue) of #140 
- [branch](https://github.com/shoshijak/dbcsr/tree/fix-build-tests) of #143

**Comment by dev-zero** [comment]
@shoshijak is it possible that you accidentally force-pushed yesterday to the `develop` branch?

The GitHub events (via the API) shows the following:

```json
[...]
  {
    "id": "9053460517",
    "type": "PushEvent",
    "actor": {
      "id": 20648084,
      "login": "shoshijak",
      "display_login": "shoshijak",
      "gravatar_id": "",
      "url": "https://api.github.com/users/shoshijak",
      "avatar_url": "https://avatars.githubusercontent.com/u/20648084?"
    },
    "repo": {
      "id": 136236635,
      "name": "cp2k/dbcsr",
      "url": "https://api.github.com/repos/cp2k/dbcsr"
    },
    "payload": {
      "push_id": 3298947934,
      "size": 0,
      "distinct_size": 0,
      "ref": "refs/heads/develop",
      "head": "2c67449e08e5edc4577f8f7cd7d28dabcd5d0472",
      "before": "572ef165e15bbc6de012e2400e25abe7bff145e0",
      "commits": [

      ]
    },
    "public": true,
    "created_at": "2019-02-12T16:09:28Z",
    "org": {
      "id": 1212942,
      "login": "cp2k",
      "gravatar_id": "",
      "url": "https://api.github.com/orgs/cp2k",
      "avatar_url": "https://avatars.githubusercontent.com/u/1212942?"
    }
  },
  {
    "id": "9042841957",
    "type": "PushEvent",
    "actor": {
      "id": 20648084,
      "login": "shoshijak",
      "display_login": "shoshijak",
      "gravatar_id": "",
      "url": "https://api.github.com/users/shoshijak",
      "avatar_url": "https://avatars.githubusercontent.com/u/20648084?"
    },
    "repo": {
      "id": 136236635,
      "name": "cp2k/dbcsr",
      "url": "https://api.github.com/repos/cp2k/dbcsr"
    },
    "payload": {
      "push_id": 3293424228,
      "size": 5,
      "distinct_size": 5,
      "ref": "refs/heads/develop",
      "head": "572ef165e15bbc6de012e2400e25abe7bff145e0",
      "before": "6b5340a192af20b03eaf3391dc9b90b41ea86f72",
      "commits": [
        {
          "sha": "8af468ac3a0e5f005c1c271e601fb2b2ae817a03",
          "author": {
            "email": "as.jak@protonmail.ch",
            "name": "shoshijak"
          },
          "message": "tests: add performance test with blocksize 23\n\nAdd a performance test using the blocksize occuring in simulations\nof the water molecule. This test features the same matrix-matrix\nmultiplication operations as the test\n`tests/QS/benchmark_DM_LS/H2O-dft-ls.inp` in CP2K",
          "distinct": true,
          "url": "https://api.github.com/repos/cp2k/dbcsr/commits/8af468ac3a0e5f005c1c271e601fb2b2ae817a03"
        },
[...]
```

so it seems that after the merge on 2019-02-11 there is a second push on 2019-02-12 from your account which reverted the HEAD of develop to an earlier commit (hence, no commits in the push).

**Comment by dev-zero** [comment]
I enabled the base protection of the `develop` branch now (no force pushes possible anymore).

You can either directly push to the DBCSR `develop` branches now (`git checkout fix/perf_issue; git push upstream develop; git checkout fix-build-tests; git rebase upstream develop; git push upstream develop` given you have configured github.com/cp2k/dbcsr.git as `upstream`) or redo the PRs.

**Comment by shoshijak** [comment]
> @shoshijak is it possible that you accidentally force-pushed yesterday to the `develop` branch?

You're right, that's probably what happened. I'm very sorry for this oversight.

> I enabled the base protection of the `develop` branch now (no force pushes possible anymore).

Thanks!

**Comment by shoshijak** [comment]
> You can either directly push to the DBCSR `develop` branches now (`git checkout fix/perf_issue; git push upstream develop; git checkout fix-build-tests; git rebase upstream develop; git push upstream develop` given you have configured github.com/cp2k/dbcsr.git as `upstream`) or redo the PRs.

done :)

**Comment by alazzaro** [comment]
@shoshijak is the current develop in sync with all your stuff? Are you planning for any other developments? If no, then we can proceed to make a new candidate release and push to CP2K...

**Comment by shoshijak** [comment]
Yes it is. I am not planning any further developments concerning the Machine Learning stuff right now. 

> If no, then we can proceed to make a new candidate release and push to CP2K...

As far as I'm concerned, that's okay. My only concern is that I'm leaving on holidays tomorrow until the 1st March, so if some troubles come up in the integration with CP2K, I won't be very reachable until March.

**Comment by alazzaro** [comment]
Great!
Well, it is incremental... First, I do a RC-1.1, then I will change the CP2K on daint to link to it, see what it does and eventually open a PR for CP2K... My assumption is that v1.1 is only for the GPU tests, i.e. Daint, so we don't need to worry about the other tests...

**Comment by shoshijak** [comment]
I understand. 
If you would like to get started on the steps you described, that is of course fine by me. 

My concern is that if some issues do show up, they will most likely be mine to fix (the biggest changes between v1 and v1.1 of DBCSR are due to me), and so your issues might be left hanging until I can give them my full attention on the 4th March. That being said, please feel free :)
